StRoot/StJetMaker/towers/StjTowerEnergyCutEt.h

```diff
@@ -12,14 +12,20 @@
 class StjTowerEnergyCutEt : public StjTowerEnergyCut {
 
 public:
-  StjTowerEnergyCutEt(double min = 0, double max = 50000.0)
-    : _min(min), _max(max) { }
+  StjTowerEnergyCutEt(double min = 0, double max = 50000.0, bool flag = true)
+    : _min(min), _max(max), _flag(flag) { }
   virtual ~StjTowerEnergyCutEt() { }
 
   bool operator()(const StjTowerEnergy& deposit)
   {
     TVector3 vec3;
```

Comment:
Creating / destroying TVector3 incurs the cost of creating / destroying TObject.  So it is better to make vec3 and vertex _static_, and use the "setMagEtaPhi" methods, rather than construcing a new object each time this operator is called.